components/ReactTable.jsx

```diff
@@ -1,10 +1,11 @@
 import React from 'react';
 import { useTable } from 'react-table';
+import { FormattedMessage, useIntl } from 'react-intl';
```

Comment:
@justin-kleid A quick note: I noticed the eslint has not been used. Can you setup the eslint quickly using the default 'strict' option? Then, try to run `next lint` and fix the linting errors, e.g. 'useIntl' is declared but its value is never read. Thanks!

Reply:
@justin-kleid I've quickly create a PR to add ESLint and Prettier https://github.com/GeoDaCenter/reactgeoda/pull/12 If you don't have time, you can review and approve my PR. Then, rebase your working branch after I merge it. Thanks!